### ISSUE TITLE
508 Accessibility: mode bar is not accessible using a keyboard

### ISSUE THREAD

**Issue by cjfreedman** [plotly/plotly.js#6940]
Section 508 Accessibility Guideline 2.1 says all functionality should be available from a keyboard.
https://www.w3.org/TR/2008/REC-WCAG20-20081211/#keyboard-operation

However, when the mode bar is displayed, there is no way to tab into it and to tab between buttons on the mode bar.  This needs to be added.
